### ISSUE TITLE
Parse fails for Metadata containing string values

### ISSUE THREAD

**Issue by donatoaz** [Skyscanner/pycfmodel#59]
Hi!

This issue surfaced while using `cfripper` to scan a json template that contains a `Metadata` key with this value:

```
"Metadata": {
    "BlueprintID#3c5b7308-d575-48bd-bd28-bce1ff16a0d9": "8.0",
    "DeploymentType": "Pipelines"
  },
```

I checked AWS' documentation on [`Metadata`](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/metadata-section-structure.html) and there seems to be no requirement that it should be modeled as a `Optional[Dict[str, Dict]]`.

Perhaps changing the value type to `Any` would be a possible alternative?

**Comment by jsoucheiron** [comment]
Would you like to submit a PR to fix it yourself?
We can do it, but we'd really like to have more commiters to the project :)

**Comment by donatoaz** [comment]
Hi jsoucheiron, thanks, I will open a first PR and will set up a local dev env to run tests. The PR is open  but I will update it with tests shortly.

**Comment by jsoucheiron** [comment]
Have you tried deploying a CF template with metadata that's only a string? The documentation is not very clear:
```
You can use the optional Metadata section to include arbitrary JSON or YAML objects that provide details about the template. For example, you can include template implementation details about specific resources, as shown in the following snippet: 
```

**Comment by oscarbc96** [comment]
@donatoaz the fix has been released on version 0.9.0 🚀 
Thanks for the contribution!
